<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProjectDialog</class>
 <widget class="QDialog" name="AddProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie projektu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nazwa Projektu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Technologia</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="technologiesLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="statusComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Czas Pracy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="workTimeSpinBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Repozytorium</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="repositoryLineEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="teamProjectCheckBox">
     <property name="text">
      <string>Projekt Zespołowy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="collaboratorsLabel">
     <property name="text">
      <string>Współautorzy (CSV)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="collaboratorsLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="responsibilitiesLabel">
     <property name="text">
      <string>Obowiązki</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="responsibilitiesLineEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
